### DISCUSSION TITLE
Accessing Workload Item through My Workspace in HomePage

### DISCUSSION THREAD

**Discussion by SanchanaMohankumar** [microsoft/Microsoft-Fabric-workload-development-sample#105] [Q&A] [answered]
Hi Team, 

Currently, on our homepage (as shown in Image 1), users can create a workload item, and underneath it, they can see "My workspace." However, as we know, a workload item can only be created when "My workspace" is configured to a Fabric capacity. If it's not configured, users receive a pop-up message (as shown in Image 2). Upon review, we observed that in image 2 the error message is misleading and does not guide users toward the correct solution.
Image 1: Workload item creation homepage
<img width="959" alt="image" src="https://github.com/user-attachments/assets/1a040efe-1ee3-458e-81cd-0268eb7db6f8">
Image 2: Pop up Error Message
<img width="530" alt="image" src="https://github.com/user-attachments/assets/fef62aa8-08b0-4488-b78a-2fc3f7552084">

Suggestion: To improve user experience, we could consider the following approaches:
1. We can redirect the user to another workspace from the homepage underneath "My workspace" if it's not configured.
2. We can rephrase the error message to provide clearer instructions and guide the user on how to configure the workspace to a Fabric capacity as currently we believe we are receiving Org.WorkloadSample.SampleWorkloadItem from the URL 

Thank you

**Comment by ramizsha** [comment, marked as answer]
Thanks for the suggestion! we will work on changing the message.
